api/config/config_perf.json

```diff
@@ -27,7 +27,8 @@
     "exportBigQueryTable": "all-of-us-workbench-test-bd.billing_data.rw_test_firecloud_view",
     "retryCount": 4,
     "bufferCapacityPerTier": {
-      "registered": 300
+      "registered": 300,
+      "controlled": 20
```

Comment:
This config change has been pushed to Perf and unblocks project creation.

Also tested: too many tiers here.  That does not cause problems.